app/src/main/AndroidManifest.xml

```diff
@@ -55,6 +55,7 @@
         <activity
             android:name=".MainActivity"
             android:configChanges="orientation|screenSize|keyboardHidden|screenLayout|smallestScreenSize"
+            android:launchMode="singleTask"
```

Comment:
I needed to add this so Tusky could receive links while it's still running in the background. I don't think it breaks anything, but let me know.